Screenshot.md

```diff
@@ -1,35 +1,32 @@
-<img src="https://user-images.githubusercontent.com/37215508/76329203-45cf5c00-6312-11ea-9b5a-62f1bd40fa03.jpg" width="288" height="500" />
-<img src="https://user-images.githubusercontent.com/37215508/76329305-68fa0b80-6312-11ea-8889-177bde47bada.jpg" width="288" height="500" />
-<img src="https://user-images.githubusercontent.com/37215508/76329500-ac547a00-6312-11ea-94fc-d771193332d6.jpg" width="288" height="500" />
-<img src="https://user-images.githubusercontent.com/37215508/76329561-be361d00-6312-11ea-8498-749c8440e5ae.jpg" width="288" height="500" />
-<img src="https://user-images.githubusercontent.com/37215508/76329746-f9385080-6312-11ea-9adb-d39798ce3f55.jpg" width="288" height="500" />
-<img src="https://user-images.githubusercontent.com/37215508/76329796-09e8c680-6313-11ea-8457-dca7f812f79d.jpg" width="288" height="500" />
-<img src="https://user-images.githubusercontent.com/37215508/76329847-19680f80-6313-11ea-94aa-a249679df064.jpg" width="288" height="500" />
-<img src="https://user-images.githubusercontent.com/37215508/76329901-284ec200-6313-11ea-9fa1-aabed403398a.jpg" width="288" height="500" />
-<img src="https://user-images.githubusercontent.com/37215508/76329949-38ff3800-6313-11ea-94c5-09daeacbf0db.jpg" width="288" height="500" />
-
-
-
-<img src="https://user-images.githubusercontent.com/37215508/76330252-9eebbf80-6313-11ea-8098-3b8f2962edb8.jpg" width="288" height="500" />
-<img src="https://user-images.githubusercontent.com/37215508/76330291-ae6b0880-6313-11ea-8625-5c55ad8b09a5.jpg" width="288" height="500" />
-<img src="https://user-images.githubusercontent.com/37215508/76330449-dbb7b680-6313-11ea-934a-08d5d0e2c8ff.jpg" width="288" height="500" />
-
-<img src="https://user-images.githubusercontent.com/37215508/76330597-073aa100-6314-11ea-9f40-9dc24a868e2c.jpg" width="288" height="500" />
-<img src="https://user-images.githubusercontent.com/37215508/76330738-3a7d3000-6314-11ea-9a4f-d6b04e433e6e.jpg" width="288" height="500" />
-<img src="https://user-images.githubusercontent.com/37215508/76330810-4ec12d00-6314-11ea-8553-e7a8bf576a9a.jpg" width="288" height="500" />
-<img src="https://user-images.githubusercontent.com/37215508/76331165-c3946700-6314-11ea-83fc-9e42a5553af3.jpg" width="288" height="500" />
-
+<img src="https://user-images.githubusercontent.com/46667021/82584052-8ddfea00-9bb1-11ea-92a9-8c83abf6322e.jpg" width="288" height="500" />
+<img src="https://user-images.githubusercontent.com/46667021/82584248-d8616680-9bb1-11ea-97b2-6a6e5129687a.jpg" width="288" height="500" />
+<img src="https://user-images.githubusercontent.com/46667021/82584330-faf37f80-9bb1-11ea-8443-bb676a99d714.jpg" width="288" height="500" />
+<img src="https://user-images.githubusercontent.com/46667021/82584418-18284e00-9bb2-11ea-94d2-5d9dedff8fb2.jpg" width="288" height="500" />
+<img src="https://user-images.githubusercontent.com/46667021/82584793-a13f8500-9bb2-11ea-847d-bb0e05279309.jpg" width="288" height="500" />
```

Comment:
Amazing organization of images. However, I feel the Markdown way is better. It helps you to organize images in tables, give a description for every image. You can refer to this: https://gist.github.com/grocky/5d75af69b4f4c10304bf. and also avoids the hardcoded way, which might differ when the orientation changes and the screen size changes. You might choose to ignore it as well :).